src/AppBundle/Service/ReportService.php

```diff
@@ -203,4 +203,31 @@ public function submit(Report $currentReport, User $user, \DateTime $submitDate)
 
         return $newYearReport;
     }
+
+    /**
+     * Set report submission for additional documents
+     *
+     * @param Report $currentReport
+     * @param User $user
+     * @param \DateTime $submitDate
+     *
+     * @return Report new year's report
+     */
+    public function submitAdditionalDocuments(Report $currentReport, User $user, \DateTime $submitDate)
```

Comment:
a unit test should be useful here. ReportServiceTest is there already

Reply:
Added Unit test